src/app/components/LocaleSwitcher/LocaleSwitcher.tsx

```diff
@@ -32,7 +32,6 @@ export default function LocaleSwitcher() {
   return (
     <Select name="locale" value={dropdownLang} onChange={languageHandler}>
       <option value="en">English</option>
-      <option value="hi">हिन्दी</option>
```

Comment:
we actually got many languages now and some are pretty much completely translated (from what can be translated right now)
Ideally we load those even dynamically here

Reply:
Yeah, possibly. On the other hand a new language would need to wait for a new release or a patch release. Not that hard to manage, right? Let's talk about it.

Reply:
not hard, but right now it's not there. let's add it.